examples/jsm/loaders/MMDLoader.js

```diff
@@ -1198,19 +1204,28 @@ class MaterialBuilder {
 				if ( material.textureIndex !== - 1 ) {
 
 					params.map = this._loadTexture( data.textures[ material.textureIndex ], textures );
+					params.map.name = material.name;
```

Comment:
Setting texture's name as material's name is a bit weird to me... Do you want to use the name for the mapping?

Reply:
uh, that's a mistake, I'll remove it.